src/_stars.css

```diff
@@ -0,0 +1,26 @@
+/* #region block: stars */
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+
+  background-image: url(./images/star.svg);
+  object-fit: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
You should add all available modifiers, not only stars--4 (stars--1, stars--2, ...)